deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -223,20 +223,22 @@ spec:
           {{- toYaml . | nindent 10 }}
         {{- end }}
       volumes:
-        - name: device-plugin
+        - name: kubelet-device-plugins-dir
           hostPath:
             path: /var/lib/kubelet/device-plugins
+            type: DirectoryOrCreate
```

Comment:
Question: Could this also be `Directory` instead?

Reply:
Updated to `Directory` to match https://github.com/GoogleCloudPlatform/container-engine-accelerators/blob/2bc811f4772f183f72db996977d684bac3fd2ae4/cmd/nvidia_gpu/device-plugin.yaml#L81C19-L81C28